src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/EvidenceUploadHandlerBase.java

```diff
@@ -693,7 +689,7 @@ CallbackResponse documentUploadTime(CallbackParams callbackParams) {
                                       PRE_TRIAL_SUMMARY_TEXT, documentDateTime -> documentDateTime.getValue().getCreatedDatetime(), defendantString);
             setCategoryIdAndRenameDoc(caseData.getDocumentSkeletonArgumentRes2(), document -> document.getValue().getDocumentUpload(), RESPONDENT_TWO_TRIAL_SKELETON,
                                       TRIAL_SKELETON_TEXT, documentDateTime -> documentDateTime.getValue().getCreatedDatetime(), defendantString);
-            setCategoryIdAndRenameDoc(caseData.getDocumentAuthoritiesRes2(), document -> document.getValue().getDocumentUpload(), RESPONDENT_TWO_PRECEDENT_H,
+            setCategoryIdAndRenameDoc(caseData.getDocumentAuthoritiesRes2(), document -> document.getValue().getDocumentUpload(), RESPONDENT_TWO_TRIAL_SKELETON,
```

Comment:
If i understand correctly this being moved from an authority/precedent type document to a skeleton type?

If so I'd assume we'd need to update the email notification text as well 

i.e. currently text would be 
protected static final String TRIAL_AUTHORITIES_TEXT = "%s - Authorities";

I guess it would now be
    protected static final String TRIAL_SKELETON_TEXT = "%s - Skeleton argument";

Reply:
they only wish to change the file location, but Authorities is still with Trail Documents>Authorities. now both skeleton argument and Authorities are in the same folder, (Trial Documents) > (Claimant/Defendant Folder) > (Skeleton argument). no need to change email notification text

Reply:
Cool, makes sense